sample/src/main/kotlin/com/adevinta/spark/sample/MainActivity.kt

```diff
@@ -20,8 +20,11 @@
  * SOFTWARE.
  */
 
+@file:SuppressLint("MaterialComposableUsageDetector")
```

Comment:
It depends what we want to do with this sample app.
Do we want to showcase something in it?
Otherwise, if it's supposed to be a shell for the Showkase app, we can remove all code from it.

Reply:
Yeah at first it'll only be a shell for the Showkase app I think

Reply:
I'll remove the code in a following review if that's ok for you.

Reply:
Good to me 👍